projects/extension/CHANGELOG.md

```diff
@@ -8,7 +8,7 @@
 
 ### Changed
 
-- Improve the UI of the popup and option pages. ([#1039](https://github.com/paritytech/substrate-connect/pull/1039), [#1061](https://github.com/paritytech/substrate-connect/pull/1061))
+- Improve the UI of the popup and options page. ([#1039](https://github.com/paritytech/substrate-connect/pull/1039), [#1061](https://github.com/paritytech/substrate-connect/pull/1061))
```

Comment:
* pages

Reply:
It's technically an HTML page, but it's not a page for the user of the extension